.travis.yml

```diff
@@ -1,28 +1,29 @@
+language: python
+
 sudo: false
-cache: pip
+cache:
+  apt: true
+  directories:
+    - $HOME/.pip-cache/
 
 addons:
   apt:
     packages:
-      - expect-dev  # provides unbuffer utility
-      - python-lxml # because pip installation is slow
-
-language: python
+      - expect-dev
+      - python-lxml
+      - python-simplejson
+      - python-yaml
 
 python:
   - "2.7"
 
 env:
   global:
-    - VERSION="8.0" TESTS="0" LINT_CHECK="0"
+  - LINT_CHECK=0 TESTS="0" VERSION="8.0" ODOO_REPO="vauxoo/odoo"
```

Comment:
On 21 October 2015 at 16:40, Lorenzo Battistini <
lorenzo.battistini@agilebg.com> wrote:

> Ok, please wait: I'm preparing a PR with other things also
>
>
Here it is https://github.com/OCA/stock-logistics-workflow/pull/170